### PR TITLE
State required versions of pytket and extensions on front page of manual.

### PR DESCRIPTION
**Pull request by cqc-alec**
This involves some minor hackery so that we only have to have the versions coded in one place when building the manual.

### CODE REVIEW

**Comment by cqc-alec** [comment]
Sed command courtesy of https://unix.stackexchange.com/questions/49377/substitute-pattern-within-a-file-with-the-content-of-other-file/49438#49438

End result:
Screenshot from 2021-03-26 06-27-30

File: build-manual [hunk 1]
```diff
@@ -1,8 +1,13 @@
 #!/bin/sh
 
-python -m pip install --pre -r requirements.txt
+python -m pip install -r requirements.txt
+
+python -m pip install --pre git+git://github.com/willsimmons1465/jupyter-sphinx.git@issue-140#egg=jupyter-sphinx
 
 cd manual/
+
+sed "s/REQUIREMENTS/$(sed -e 's/[\&/]/\\&/g' -e 's/$/\\n/' ../requirements.txt | tr -d '\n')/" index-template.rst > index.rst
```
Comment (yao-cqc) [inline build-manual:9]: Can we remove `index.rst` after the doc build?

Comment (cqc-alec) [reply build-manual:9]: Yes we should...

**Review by yao-cqc — APPROVED** [review]
Thanks Alec! This is very nice.

**Pull request merged**